userdash.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Bright-Events</title>
+    <link rel="stylesheet" type="text/css" href="stylesheet/userdash.css">
+  </head>
+  <body>
+    <div class="topnav">
+      <a href="#">My Events</a>
```

Comment:
Include the design of how the the dashboard will look like or events will appear. Just like what you have in your wireframes designs.